src/lib/interactionManager/credentialRequestFlow.ts

```diff
@@ -41,6 +42,16 @@ export class CredentialRequestFlow extends Flow {
   }
 
   private async handleCredentialRequest(request: CredentialRequest) {
+    this.state.constraints.push(request)
+    return true
+  }
+
+  private async handleCredentialResponse(token: CredentialResponse) {
+    this.state.providedCredentials.push(token)
+    return token.satisfiesRequest(this.state.constraints[-1])
+  }
+
+  public async getAvailableCredentials(request: CredentialRequest) {
```

Comment:
is sort of "static", except for needing ctx, as it does not depend on flow state at all. should probs be moved out of the flow class